packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -64,22 +64,45 @@ export interface ITree extends IChannel {
 	): TreeView<TRoot>;
 }
 
+/**
+ * Options when schematizing a tree.
+ * @public
+ */
+export interface ITreeConfigurationOptions {
+	/**
+	 * If `true`, the tree will validate new content against its stored schema at insertion time
```

Comment:
We should probably note in a `@remarks` block that there is potential performance cost to this.